src/main/java/com/theelixrlabs/healthcare/dto/DoctorDto.java

```diff
@@ -0,0 +1,36 @@
+package com.theelixrlabs.healthcare.dto;
+
+import com.theelixrlabs.healthcare.constants.DoctorConstants;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+/**
+ * Data transfer object representing doctor information.
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@Data
+@SuperBuilder
```

Comment:
Do you actually need @Getter and @Setter when you have add @Data ?

Reply:
No. there is no need of additional @getter @setter. I removed those.